metals/src/main/scala/scala/meta/internal/metals/Compilers.scala

```diff
@@ -684,19 +724,29 @@ class Compilers(
       params: TextDocumentPositionParams,
       token: CancelToken,
   ): Future[ju.List[TextEdit]] =
-    withPCAndAdjustLsp(params) { (pc, pos, adjust) =>
-      pc.inlineValue(CompilerOffsetParamsUtils.fromPos(pos, token))
-        .asScala
+    withPCAndAdjustLsp(params) { (pc, pos, adjust, buildTargetId) =>
+      pc.inlineValue(
+        CompilerOffsetParamsUtils.fromPos(
+          pos,
+          token,
+          outlineFilesProvider.getOutlineFiles(buildTargetId),
+        )
+      ).asScala
         .map(adjust.adjustTextEdits)
     }.getOrElse(Future.successful(Nil.asJava))
 
   def documentHighlight(
       params: TextDocumentPositionParams,
       token: CancelToken,
   ): Future[ju.List[DocumentHighlight]] = {
-    withPCAndAdjustLsp(params) { (pc, pos, adjust) =>
-      pc.documentHighlight(CompilerOffsetParamsUtils.fromPos(pos, token))
-        .asScala
+    withPCAndAdjustLsp(params) { (pc, pos, adjust, buildTargetId) =>
```

Comment:
Maybe we could add `getBuildTargetId(): Optional[String]` and this would avoid the changes in the file

Reply:
It won't really avoid the changes, we still have to pass `outlineFilesProvider.getOutlineFiles(buildTargetId)`

Reply:
But we could do `outlineFilesProvider.getOutlineFiles(pc.getBuildTargetId)` or similar no?

Reply:
yes